app/src/main/java/us/shandian/giga/ui/fragment/MissionsFragment.java

```diff
@@ -193,7 +193,7 @@ public boolean onOptionsItemSelected(MenuItem item) {
                 prompt.setMessage(R.string.confirm_prompt);
                 // Intentionally misusing button's purpose in order to achieve good order
                 prompt.setNegativeButton(R.string.clear_download_history, (dialog, which) -> mAdapter.clearFinishedDownloads(false));
```

Comment:
Move your cursor to `R.string.clear_download_history` and press `STRG + B`. It should jump to a `string.xml` file. Translations should be in this file and then referenced like this.

Reply:
What is `STRG`? You mean `Ctrl`?

Reply:
ehh yes, strg is german for ctrl